test/functional/desired.js

```diff
@@ -1,59 +1,69 @@
 import _ from 'lodash';
-import '../env';
-
 
 const uiautomator2ServerLaunchTimeout = (process.env.TRAVIS || process.env.CI) ? 60000 : 20000;
 const uiautomator2ServerInstallTimeout = (process.env.TRAVIS || process.env.CI) ? 120000 : 20000;
 
 const ADB_EXEC_TIMEOUT = (process.env.TRAVIS || process.env.CI) ? 60000 : 20000;
 
-const GENERIC_CAPS = {
-  deviceName: 'Android',
-  platformName: 'Android',
-  uiautomator2ServerLaunchTimeout,
-  uiautomator2ServerInstallTimeout,
-  automationName: 'uiautomator2',
-  adbExecTimeout: ADB_EXEC_TIMEOUT,
-  ignoreHiddenApiPolicyError: true,
-};
+function deepFreeze (object) {
```

Comment:
this seems like the type of thing that could be in appium support rather than implementing in multiple projects

Reply:
will do as the next step